include/amici/model.h

```diff
@@ -1704,6 +1740,12 @@ class Model : public AbstractModel, public ModelDimensions {
      * checked for finiteness
      */
     bool always_check_finite_ {false};
+    
+    /** indicates whether sigma residuals are to be added for every  */
+    bool sigma_res_ {false};
+    
+    /** offset to ensure positivity of sigma residuals, only has an effect when `sigma_res_` is `true`  */
+    realtype min_sigma_ {50.0};
```

Comment:
Just as a remark:
The higher the offset, the weaker the contribution of this term to the gradient will be, right? The square root function flattens out for higher values... So, I my gut feeling tells me that this offset term influences the parameter estimation process somehow. Therefore, I would suggest a lower value than 50, which is however still high enough to not cause problems for typical applications. I think already 10 should do the job here...

Reply:
I agree that the offset should have an influence. No idea about practically typical values (ok, log(2*pi*1e-8^2)=-35, one could look at typical bounds in the benchmark collection).

Reply:
Note that we do not account for the `2pi` factor here (we could). The gradient itself is not influenced by C, but `sres` is and the FIM is. Probably the smartest way to go about this is set some value based on bounds for sigmas in pypesto.